Index.html

```diff
@@ -5,11 +5,190 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+
+    <link rel="stylesheet" href="/css/Main.css">
     <title>Profolio | Home</title>
 </head>
 <body>
-    <main>
+    <main class="Main_Container">
+        <nav>
+            <p>John Doe</p>
+            <i class="fas fa-bars"></i>
+        </nav>
+        <section class="Home_Banner">
+            <div class="Home_Title">
+                <h1>Laura Jong</h1>
+                <p>Sofware Developer</p>
+            </div>
+            <div class="Home_Text">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+        </section>
+        <section class="Skills_Main">
+            <div class="Skills_Box">
+                <div class="Skills_Titles">
+                    <h2>Langauage</h2>
+                </div>
+                <div class="Skills_List">
+                    <p>Javascript</p>
+                    <p>Ruby</p>
+                    <p>HTML</p>
+                    <p>Css</p>
+                </div>
+            </div>
+            <div class="Skills_Box">
+                <div class="Skills_Titles">
+                    <h2>Framework</h2>
+                </div>
+                <div class="Skills_List">
+                    <p>Boostrap</p>
+                    <p>Ruby on Rails</p>
+                    <p>RSPec</p>
+                    <p>CopyBars</p>
+                    <p>Selenium</p>
+                </div>
+            </div>
+            <div class="Skills_Box">
+                <div class="Skills_Titles">
+                    <h2>Skills</h2>
+                </div>
+                <div class="Skills_List">
+                    <p>Codekit</p>
+                    <p>Github</p>
+                    <p>Codepen</p>
+                    <p>Gitlab</p>
+                    <p>Terminal</p>
+                </div>
+            </div>
+        </section>
+
+        <section class="Social_Icon">
+            
+            <div class="Social_List">
+                <a href="#"><i class="fab fa-angellist"></i></a>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+                <a href="#"><i class="fab fa-linkedin-in"></i></a>
+                <a href="#"> <i class="fab fa-github"></i></a>
+            </div>
```

Comment:
When we are dealing with multiple links like in this instance, for better practices and semantics, we should group them in a ul tag and each list item(icons) inside the ul element should be as a li 😁